app/services/builders/chartkick/development_metrics.rb

```diff
@@ -51,7 +51,7 @@ def defect_escape_rate_entities
         end
 
         def project_has_jira_board_associated?(project_id)
-          ::Project.find(project_id).jira_project.present?
+          ::Project.find(project_id).jira_project?
```

Comment:
I prefer to have `.present?` here instead of creating a method in the model

Reply:
Done!